tools/coq_makefile.ml

```diff
@@ -124,7 +124,7 @@ let read_whole_file s =
     close_in ic;
     Buffer.contents b
 
-let quote s = if String.contains s ' ' || String.equal s "" then "'" ^ s ^ "'" else s
+let quote s = if String.contains s ' ' || s = "" then "'" ^ s ^ "'" else s
 
```

Comment:
I'd recommend using `String.is_empty`.

Reply:
Ah, that’s better of course. I did not find it on https://caml.inria.fr/pub/docs/manual-ocaml/libref/String.html.

Reply:
Hm, doesn't work here:
```
File "tools/coq_makefile.ml", line 127, characters 42-57:
Error: Unbound value String.is_empty
```